# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/xtris/xlibs.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN xtris/Makefile xtris-linux/Makefile
--- xtris/Makefile	1997-12-14 19:43:35.000000000 +0100
+++ xtris-linux/Makefile	2003-06-13 21:26:50.000000000 +0200
@@ -23,7 +23,7 @@
 # EXTRALIBS = -lnsl -lsocket 
 
 # specify X11 libdir if your system needs it
-# XLIBDIR = -L/usr/X11/lib
+XLIBDIR = -L/usr/X11/lib
 
 
 ###########################################################################
